infrastructure/terraform/modules/backend-api/spec.tmpl.json

```diff
@@ -777,7 +777,8 @@
           "VALIDATION_FAILED",
           "VIRUS_SCAN_FAILED",
           "WAITING_FOR_PROOF",
-          "PROOF_AVAILABLE"
+          "PROOF_AVAILABLE",
+          "TEMPLATE_PROOF_APPROVED"
```

Comment:
It feels strange having this on the API spec when this isn't a status on the backend. The API will never return or accept this status. I know its used to generate the types. But I wonder if there's a that the status change can be shifted further left or further right. 

Further left would be something like we make the API return this status (we wouldn't have to change in the database). Maybe we update the Zod parser so that it transforms the status from SUBMITTED to TEMPLATE_PROOF_APPROVED.

Or further right would be something like ... I will comment in the appropriate place 😅